### ISSUE TITLE
bug: Event NPC become duplicated  

### ISSUE THREAD

**Issue by ZerinSad** [AscensionGameDev/Intersect-Engine#1791]
### Description

When the npc created by event walks to the border of some other map, he gets duplicated for some reason, i dont know if this is happening only in my project, that is very old database, since 2018 i think, but the image of my forum post tells everything. 

https://www.ascensiongamedev.com/topic/6821-event-npc-become-duplicated/

[https://ascensiongamedev.com/resources/filehost/499b21841f309fc62155c0c5061b5d4a.mp4](https://www.ascensiongamedev.com/resources/filehost/499b21841f309fc62155c0c5061b5d4a.mp4)

### Steps to Reproduce

in editor, create a npc event and puts he to walk often
create the event close to the border of the map
wait 5min and go at this map, with your character, in game.
see the bug happening

### Version with bug

7.1.120

### Last version that worked well

7.1.119

### Affected platforms

Windows

### Did you find any workaround?

create the event npc in the middle of the map and put him to walk not very often, but still bugging 

### Duplicate Bug Check

- [X] This bug report is not a duplicate to the best of my knowledge.

**Comment by ValeFolken** [comment]
![relicdogsplit](https://github.com/AscensionGameDev/Intersect-Engine/assets/127902267/46a689d6-eea4-4368-b1fb-fdcea9069037)

**Comment by WeylonSantana** [comment]
events are programmed not to leave the map, apparently some people report that events are exceeding the map limit, are we missing something here?

**Comment by naziozeno** [comment]
yeah i can confirm this bug, events are leaving the map, whenever the npc pass to another linked map a close appear, the original event continue moving and the clone stay frozen with no interactions.

**Comment by ValeFolken** [comment]
In the photo posted, the vertical line of the map goes down along the left side of the house.  The dog's spawn location is just to the left of the haystack, which confirms travel across the map borders

**Comment by NikoCummings** [comment]
Alright, I replicated this bug on the latest version of vanilla intersect. Below is a copy so anyone can check it out for themselves. 
https://www.ascensiongamedev.com/resources/filehost/1465d3b347bce94ef24639790553f1ea.rar

Don't mind all the NPCs, resources, and attributes I slapped all over the map. I was trying to recreate the "busy" map I was encountering the bug on. I was unable to replicate the bug until I enabled diagonal movement via the server config file. Once I did this and logged back into the game the events started making copies of themselves. It does happen near the edge of the maps and the duplicated event does appear on the maps touching the main map. The duplicated events are generally just paralyzed there along the border of the two maps.
